### PR TITLE
[one-cmd] Add section, input_path, output_path for one-init

### PR DESCRIPTION
**Pull request by yunjayh**
This commit enables one-init to add sections and
input_path and output_path of each section.

Signed-off-by: yunjay-hong <yunjay.hong@samsung.com>

---

From #9619

### CODE REVIEW

File: compiler/one-cmds/one-init [hunk 1]
```diff
@@ -253,6 +326,25 @@ def _generate():
...
+        _, ext = os.path.splitext(args.input_path)
+
+        # ext would be, e.g., '.tflite' or '.onnx'.
+        # Note: when args.input_path does not have an extension, e.g., '/home/foo'
+        # ext after os.path.splitext() is '' and ''[1:] is still ''.
+        ext = ext[1:]
+        if ext in ["tflite", "onnx"]:
+            return ext
+        else:
+            parser.error(f'following file extensions are supported: ".onnx" ".tflite"')
```
Comment (yunjayh) [inline compiler/one-cmds/one-init:343]: **FYI** These lines are added from #9619 to resolve https://github.com/Samsung/ONE/pull/9619#discussion_r956806115.

Comment (seanshpark) [inline compiler/one-cmds/one-init:343]: Q) What about TensorFlow models? maybe add TODO comments?

File: compiler/one-cmds/one-init [hunk 2]
```diff
@@ -253,6 +326,25 @@ def _generate():
...
+def _get_model_type(parser, args):
+    if _utils._is_valid_attr(args, 'model_type'):
+        return args.model_type
+
+    if _utils._is_valid_attr(args, 'input_path'):
+        _, ext = os.path.splitext(args.input_path)
+
+        # ext would be, e.g., '.tflite' or '.onnx'.
+        # Note: when args.input_path does not have an extension, e.g., '/home/foo'
+        # ext after os.path.splitext() is '' and ''[1:] is still ''.
```
Comment (yunjayh) [inline compiler/one-cmds/one-init:338]:
```suggestion
        # ext after os.path.splitext() is '' and ''[1:] is still ''.
        # TODO support tensorflow model
```

File: compiler/one-cmds/one-init (hunk 1 above)
Comment (yunjayh) [reply compiler/one-cmds/one-init:344]: Added! https://github.com/Samsung/ONE/pull/9813/files#r983157894

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Comment by llFreetimell** [comment]
@nnfw-bot test nncc-release

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**